storage/ndb/src/kernel/blocks/dbtup/DbtupTrigger.cpp

```diff
@@ -1909,7 +1936,27 @@ bool Dbtup::readTriggerInfo(TupTriggerData *const trigPtr,
           get_copy_tuple(&req_struct->prevOpPtr.p->m_copy_tuple_location);
     }
 
-    if (regTabPtr->need_expand(disk)) prepare_read(req_struct, regTabPtr, disk);
+    if (regTabPtr->need_expand(disk)) {
```

Comment:
This code is required, but given that the above code is removed, this means that there is no TODO left.
The 2 prepare_read calls are one for after value and one for before value.

The only case where we need worry about extra disk page is with SUBSCRIPTION_BEFORE, thus with
the binlog case. And here the below code is required.

Reply:
Also don't forget to update Copyright headers of files changed